packages/storybook/collection.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "Nx Cypress",
+  "version": "0.1",
+  "schematics": {
+    "ng-add": {
+      "factory": "./src/schematics/ng-add/ng-add",
+      "schema": "./src/schematics/ng-add/schema.json",
+      "description": "Add storybook configuration to the workspace",
+      "hidden": true
+    },
+    "storybook-configure": {
```

```suggestion
    "configuration": {
```

It feels sort of redundant to have a `storybook-` prefix in a collection of `storybook` schematics.